lib/components/T3MediaFile/type/Video.vue

```diff
@@ -1,22 +1,20 @@
-<template functional>
+<template>
   <video
-    v-bind="data.attrs"
-    :class="[data.class, data.staticClass]"
+    v-bind="$attrs"
```

Comment:
is there a need to use v-bind="$attrs" since we don't use inheritAttrs:false here ?